pkg/controller/routes.go

```diff
@@ -56,6 +56,7 @@ func routes(router *router.Router, registryTransport http.RoundTripper) {
 	appRouter.HandlerFunc(appdefinition.ReadyStatus)
 	appRouter.HandlerFunc(appdefinition.UpdateGeneration)
 	appRouter.HandlerFunc(appdefinition.AddAcornProjectLabel)
+	appRouter.HandlerFunc(appdefinition.NetworkPolicy)
```

Comment:
`UpdateGeneration` should be the last handler here. Let's move the last two up (I know you didn't add one of them, but move it too).